polyfill.js

```diff
@@ -14,8 +14,10 @@ if (typeof Array.prototype.flatMap !== 'function') {
     // https://tc39.github.io/proposal-flatMap/#sec-Array.prototype.flatMap
     value: function flatMap(callback, thisArg = undefined) {
       const O = ES.ToObject(this);
+      // Could ES.IsCallable check callback, but it's better to let it throw a runtime error
```

Comment:
Writing a throw here would require writing an error message that is probably less informative than whatever the engine can come up with for the call inside flattenIntoArray.

However, there indeed needs to be a way to throw if a falsy but not undefined value is passed, which flattenIntoArray did not account for. Maybe hangs the check there to !== undefined.